ovn-org/ovn-kubernetes
# Metric: Add egress firewall rules metric

Add a metric to count the number of egress firewall rules per namespace.
This metric will facilitate an understanding if egress firewall is being used and also how heavily it is being used across all namespaces. It will give us understanding into the total rules per namespace.

We allow one egress firewall CR per namespace.
This metric is not being exposed by kube-state-metrics.

Signed-off-by: Martin Kennelly <mkennell@redhat.com>

**- How to verify it**
1) Apply an egress firewall CR with X amount of rules.
2) Curl metrics endpoint and grep for 'num_egress_firewall_rules'. Expect to see one entry with label namespace and X value.

**- Description for the changelog**
None
